### PR TITLE
Update logging chart to upstream 3.9.0

### PR DESCRIPTION
**Pull request by nickgerace**
Relevant Issue: https://github.com/rancher/rancher/issues/30917
Image Mirror PR: https://github.com/rancher/image-mirror/pull/65

### CODE REVIEW

**Comment by nickgerace** [comment]
Notes for reviewers...
- `serviceMonitor` is now disabled by default; matching our `values.yaml`
- `createCustomResource` is now disabled by default; matching our `values.yaml`
- `.values.rbac.psp` has been added. This might be worth looking into relative to https://github.com/rancher/charts/pull/951

File: packages/rancher-logging/generated-changes/patch/values.yaml.patch [hunk 1]
```diff
@@ -35,13 +26,19 @@
...
-   serviceMonitor:
--    enabled: false
-\ No newline at end of file
-+    enabled: false
+@@ -51,7 +56,7 @@
+     labels: {}
+ 
+ rbac:
+-  enabled: true
++  enabled: false
```
Comment (aiyengar2) [inline packages/rancher-logging/generated-changes/patch/values.yaml.patch:34]: Why are we choosing to disable this?

File: packages/rancher-logging/generated-changes/patch/values.yaml.patch [hunk 2]
```diff
@@ -77,16 +74,16 @@
...
-+    tag: 1.6.4
++    tag: 1.6.10
 +  fluentbit_debug:
 +    repository: rancher/fluent-fluent-bit
-+    tag: 1.6.4-debug
++    tag: 1.6.10-debug
 +  fluentd:
 +    repository: rancher/banzaicloud-fluentd
-+    tag: v1.11.5-alpine-1
++    tag: v1.11.5-alpine-9
```
Comment (aiyengar2) [inline packages/rancher-logging/generated-changes/patch/values.yaml.patch:86]: Double checked that these images exist; LGTM

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: packages/rancher-logging/generated-changes/patch/values.yaml.patch (hunk 1 above)
Comment (nickgerace) [reply packages/rancher-logging/generated-changes/patch/values.yaml.patch:34]: It did not exist prior and we have our own overlay. However, I will double check to see what the enabled rbac entails

File: packages/rancher-logging/package.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
-url: https://kubernetes-charts.banzaicloud.com/charts/logging-operator-3.8.2.tgz
-packageVersion: 02
+url: https://kubernetes-charts.banzaicloud.com/charts/logging-operator-3.9.0.tgz
+packageVersion: 01
 releaseCandidateVersion: 01
```
[the remark this answers is not in the document]
Comment (nickgerace) [reply packages/rancher-logging/package.yaml:3]: Will do

File: packages/rancher-logging/generated-changes/patch/values.yaml.patch (hunk 1 above)
Comment (nickgerace) [reply packages/rancher-logging/generated-changes/patch/values.yaml.patch]: Nevermind. I was off and nothing changed. Fixed.

**Review by brendarearden — APPROVED** [review]
LGTM

**Review by aiyengar2 — APPROVED** [review]
LGTM

File: packages/rancher-logging/package.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
-url: https://kubernetes-charts.banzaicloud.com/charts/logging-operator-3.8.2.tgz
-packageVersion: 02
-releaseCandidateVersion: 01
+url: https://kubernetes-charts.banzaicloud.com/charts/logging-operator-3.9.0.tgz
+packageVersion: 00
+releaseCandidateVersion: 00
```
Comment (cbron) [inline packages/rancher-logging/package.yaml:3]: Hmm, back to the 000 problem. Oh well.

**Review by cbron — COMMENTED** [review]
Are we keeping logging in a single line for now ? Not splitting folders for versions.

**Comment by nickgerace** [comment]
@cbron: yes, we are just sticking to the current package name.

**Pull request merged**